### PR TITLE
update 6.0.1 change log

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Draft 6.0.1 change log

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [NA] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [NA] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [NA] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [NA] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Review by bergmeister — CHANGES_REQUESTED** [review]
There should be no space character between the '@' and 'iSazonov'

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v6.0.1 - 2018-01-10
+
+### Engine updates and fixes
+
+- Update PowerShell to use `2.0.5` dotnet core runtime. (#5903)
```
[the remark this answers is not in the document]
Comment (bergmeister) [reply CHANGELOG.md]: Except that there should be no space character.

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@TravisEz13 There is one spelling error from the last travis-ci run:
```
The command "pwsh -File tools/travis.ps1" exited with 1.
1.41s$ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then mdspell '**/*.md' '!**/Pester/**/*.md' --ignore-numbers --ignore-acronyms --report; fi
    CHANGELOG.md
        3 | ## v6.0.1 - 2018-01-19 
>> 1 spelling error found in 65 files
```

**Review by bergmeister — CHANGES_REQUESTED** [review]
@SteveL-MSFT  I think at least a link to the security vulnerabily affecting 6.0.0 (that caused the .Net Core patch to be taken into this patch) should be given with some guidance which kind of PowerShell usage it affects.

**Comment by TravisEz13** [comment]
restarted Linux CI due to bootstrap failure

**Comment by bergmeister** [comment]
Question: Is the patch optional or does this mean that the support for 6.0.0 gets dropped. The modern lifecycle doc [here](https://docs.microsoft.com/en-us/powershell/scripting/PowerShell-Core-Support?view=powershell-6) does not talk about patch policy. This question came up because PSSA is going to get the latest .net core runtime version that 6.0.1 has (and will therefore might get incompatible with 6.0.0), assuming that 6.0.0 gets deprecated (hence this question)

**Comment by SteveL-MSFT** [comment]
@bergmeister we'll have an Announcements repo with the detailed information similar to how dotnet is doing it.  Regarding support, @joeyaiello is working on an update to the support doc to include patched versions.

**Review by SteveL-MSFT — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v6.0.1 - 2018-01-19
```
Comment (daxian-dbw) [inline CHANGELOG.md]: Are we going to change this date?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+### Engine updates and fixes
+
+- Update PowerShell to use `2.0.5` dotnet core runtime and packages. (#5903, #5961) (Thanks @iSazonov!)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: One more update needs to be added for #5961

Comment (TravisEz13) [reply CHANGELOG.md]: What additional is needed for 5961?

Comment (daxian-dbw) [reply CHANGELOG.md]: Never mind. I didn't see the `#5961` was mentioned.

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**